add a link to community call records

Issif opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines in the [CONTRIBUTING.md](https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md) file in the Falco repository.
2. Please label this pull request according to what type of issue you are addressing.
3. Please add a release note!
4. If the PR is unfinished while opening it specify a wip in the title before the actual title, for example, "wip: my awesome feature"
-->

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:

This PR adds a link to community call records:

![image](https://user-images.githubusercontent.com/4520100/154964779-1164c092-d616-4021-92a2-ce9edb20bf46.png)

**Which issue(s) this PR fixes**:

<!--
Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->

Fixes #

**Special notes for your reviewer**:

leogr reviewed (commented):
Is this :point_down: empty file needed?
[.hugo_build.lock](https://github.com/falcosecurity/falco-website/pull/557/files#diff-4b60d2826085e792088186acd4de31d5f4639e950bc66c688f9b36eb38fe79ed)

poiana commented:
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/falcosecurity/falco-website/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [9d17c4e](https://github.com/falcosecurity/falco-website/commits/9d17c4ef41a11a9dd50677c11177e50eb43baf08) Merge branch 'falcosecurity:master' into master

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

Issif commented:
> Is this point_down empty file needed? [.hugo_build.lock](https://github.com/falcosecurity/falco-website/pull/557/files#diff-4b60d2826085e792088186acd4de31d5f4639e950bc66c688f9b36eb38fe79ed)

Fixed, and I added an item in  .gitignore for avoiding other issue like this one :+1:

poiana commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/falcosecurity/falco-website/pull/557#" title="Author self-approved">Issif</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **leogr** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @leogr` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://prow.falco.org/command-help?repo=falcosecurity%2Ffalco-website).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["leogr"]} -->

poiana commented:
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/falcosecurity/falco-website/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [9d17c4e](https://github.com/falcosecurity/falco-website/commits/9d17c4ef41a11a9dd50677c11177e50eb43baf08) Merge branch 'falcosecurity:master' into master

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
